src/index.ts

```diff
@@ -108,11 +123,13 @@ export class SDK {
                 onSelectedToolChanged: this.subscriber.onSelectedToolChanged,
                 onUndoStateChanged: this.subscriber.onUndoStateChanged,
                 onSelectedLayoutFramesChanged: this.subscriber.onSelectedLayoutFramesChanged,
+                onSelectedTextStyleChanged: this.subscriber.onSelectedTextStyleChanged,
             },
             this.setConnection,
             this.config.editorId,
         );
         this.editorAPI = connection?.promise.then((editorAPI) => {
+            console.log(editorAPI, 'HEERE WE GOOO');
```

Comment:
wild log appeared

Reply:
Done 3b231437acc17ad1deb4db49d47128d9ebdca9a9

Reply:
can you remove it?

Reply:
Sorry,done 86f08e5b307652d6007f50c05e316e9044767dc0